src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/components/DataBlockDetailsForm.tsx

```diff
@@ -146,6 +148,11 @@ const DataBlockDetailsForm = ({
                 id="data-block-title"
                 name="title"
                 label="Table title"
+                onChange={e => {
+                  e.preventDefault();
+                  // eslint-disable-next-line no-unused-expressions
+                  onTitleChange && onTitleChange(e.currentTarget.value);
```

Comment:
Don't be lazy. Write this properly as an if and you don't have to put an eslint-disable